docs/source/requests.md

```diff
@@ -1,52 +1,81 @@
 ---
-title: POST and GET format
-description: How to send requests to Apollo Server over HTTP.
+title: Operation request format
+sidebar_title: Request format
+description: How to send requests to Apollo Server over HTTP
 ---
 
-Apollo Server accepts both GET and POST requests.
+> By default, almost every GraphQL client library takes care of sending operations in a format that Apollo Server supports. This article describes that format.
+
+Apollo Server accepts queries and mutations sent as POST requests. It also accepts queries sent as GET requests.
 
 ## POST requests
 
-Apollo Server accepts POST requests with a JSON body. A valid request must contain either a `query` or an `operationName` (or both, in case of a named query), and may include `variables.` Here's an example for a valid body of a post request:
+Apollo Server accepts POST requests with a JSON body. A valid request contains a `query` field, along with optional `variables` and an `operationName` (if `query` contains multiple possible operations).
 
-```js
-{
-  "query": "query aTest($arg1: String!) { test(who: $arg1) }",
-  "operationName": "aTest",
-  "variables": { "arg1": "me" }
+Let's say we want to execute the following query:
+
+```graphql
+query GetBestSellers($category:ProductCategory) {
+  bestSellers(category: $category) {
+    title
+  }
 }
 ```
 
-Variables can be an object or a JSON-encoded string. I.e. the following is equivalent to the previous query:
+ Here's an example of a valid POST request body for that query:
 
-```js
+```json
 {
-  "query": "query aTest($arg1: String!) { test(who: $arg1) }",
-  "operationName": "aTest",
-  "variables": "{ \"arg1\": \"me\" }"
+  "query":"query GetBestSellers($category:ProductCategory){bestSellers(category: $category){title}}",
+  "operationName": "GetBestSellers",
+  "variables": { "category": "BOOKS" }
 }
 ```
 
-### Batching
+The `variables` field can be an object or a JSON-encoded string. The following is equivalent to the previous body:
```

Comment:
I think we should document this alternative as legacy or not recommended, or maybe even not document it at all. There's not really a good reason to support it other than backwards compatibility (the code was sloppily written so that GET and POST went through the same code paths).